### PR TITLE
Add metametrics check in onboarding-opt-in spec

### PR DESCRIPTION
**Pull request by cortisiko**
**Description**

_Write a short description of the changes included in this pull request._

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

File: e2e/specs/start-exploring.spec.js [hunk 1]
```diff
@@ -45,22 +45,25 @@ describe('Start Exploring', () => {
...
 		await OnboardingView.tapCreateWallet();
 
 		await MetaMetricsOptIn.isVisible();
-		await MetaMetricsOptIn.tapAgreeButton();
+		await MetaMetricsOptIn.tapNoThanksButton();
 
 		await CreatePasswordView.isVisible();
+	});
+
+	it('should be able to opt-out of the onboarding-wizard', async () => {
```
Comment (gantunesr) [inline e2e/specs/start-exploring.spec.js:57]: Looks like there are some tests with the same name, is it OK for it to be this way?

**Review by gantunesr — COMMENTED** [review]
Left a small concern

File: e2e/specs/start-exploring.spec.js (hunk 1 above)
Comment (cortisiko) [reply e2e/specs/start-exploring.spec.js]: this is a good catch. ty @gantunesr. Fixed!

File: e2e/specs/onboarding-wizard-opt-in.spec.js [hunk 2]
```diff
@@ -84,65 +83,52 @@ describe('Onboarding wizard opt-out', () => {
...
-	// 	// Check that the onboarding wizard is present
-	// 	await OnboardingWizardModal.isVisible();
-	// 	await OnboardingWizardModal.tapTakeTourButton();
-
-	// 	// Tap on Skip Tutorial
-	// 	await OnboardingWizardModal.tapSkipTutorialButton();
-	// 	await OnboardingWizardModal.isNotVisible();
-	// });
+
+	it('should dismiss the onboarding wizard', async () => {
```
Comment (gantunesr) [inline e2e/specs/onboarding-wizard-opt-in.spec.js:111]: This one also looks to be repeated. Sorry for not pointing it out earlier 😅

**Review by gantunesr — APPROVED** [review]
Left another small comment.

LGTM

File: e2e/specs/onboarding-wizard-opt-in.spec.js (hunk 2 above)
Comment (cortisiko) [reply e2e/specs/onboarding-wizard-opt-in.spec.js:111]: fixed!

**Pull request merged**